CHANGELOG.md

```diff
@@ -6,7 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+- `binarySearchThreshold` option is now deprecated, as every search of sorted data always uses binary search.
```

Comment:
Can you add `deprecated` tag to the `binarySearchThreshold` option description?
ie `* @deprecated Every search of sorted data always uses binary search`

Reply:
in the Config?

Reply:
yes, it will be displayed in the API reference